Krypton/Notify.swift

```diff
@@ -182,6 +186,14 @@ class Notify {
             self.noteMutex.unlock()
             
             log("pushing note with id: \(noteId)")
+            
+            let localRequest = LocalNotificationAuthority.VerifiedLocalRequest(alertText: content.body,
+                                                                               request: request,
+                                                                               sessionID: session.id,
+                                                                               sessionName: session.pairing.displayName)
+            
+            content.userInfo = (try? LocalNotificationAuthority.createSignedPayload(for: localRequest)) ?? [:]
```

Comment:
should an error here instead be displayed as an error in the notification content?

Reply:
👍